tests/unit/files/file_systems/test_file_systems.py

```diff
@@ -24,18 +24,22 @@ def test_create_zfs_file_system(self, mock_send_request):
         """Test creating a zfs sends a request"""
         mock_send_request.return_value = mock.Mock(headers={"Content-Type": "application/json"}, status_code=201)
 
-        Files(self.test_profile).create_zfs_file_system(
-            "file_system_name", {"perms": 100, "cylsPri": 16777213, "cylsSec": 16777215}
-        )
-        mock_send_request.assert_called_once()
+        try: 
+            Files(self.test_profile).create_zfs_file_system(
+                "file_system_name", {"perms": 100, "cylsPri": 16777213, "cylsSec": 16777215}
+            )
+        except:    
+            mock_send_request.assert_called_once()
```

Comment:
I think we should always assert that `mock_send_request` is called, not just in the `except` block. Also, is the try/except necessary? It seems like there might have been an error encountered while running these tests, which could explain the try/except, but this test seems intended for validating a successful outcome.

I have the same request for the other tests changed in this test file:
- 39-42
- 65-68
- 75-78
- 89-92